spring-pulsar/src/main/java/org/springframework/pulsar/support/PulsarHeaders.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.support;
+
+/**
+ * Pulsar specific message headers.
+ *
+ * @author Soby Chacko
+ */
+public abstract class PulsarHeaders {
+
+	/**
+	 * The prefix for Pulsar headers.
+	 */
+	public static final String PREFIX = "pulsar_";
+
+	/**
+	 * The prefix for the message.
+	 */
+	public static final String PULSAR_MESSAGE = PREFIX + "message_";
```

Comment:
Wdyt about shortening from `message_` to `msg_`?

Reply:
Any particular reason for the abbreviation? I thought `messge` avoids any confusion.